src/BlockTypes/LegacyTemplate.php

```diff
@@ -180,4 +191,26 @@ protected function render_archive_product() {
 		wp_reset_postdata();
 		return ob_get_clean();
 	}
+
+	/**
+	 * Get HTML markup with the right classes by attributes.
+	 *
+	 * @param string $content Block content.
+	 * @param array  $block Parsed block data.
+	 * @return string Rendered block type output.
+	 */
+	public function get_markup_with_classes_by_attributes( string $content, array $block ) {
```

Comment:
This callback should run only for the `LegacyTemplate` block. Currently, it runs for every block.

Reply:
So do you suggest adding an if condition to check if is it a `LegacyTemplate`?

Reply:
Yes.